atlp-rwanda/commanders-rn-medica
# Feature: Chats , calls History  implementation

What does this PR do?
Display Message chats, Video calls, voice calls history screen for appointments that happened between a patient and a doctor.It also shows completed appointments

Description of Task to be completed?

Fetch booked appointment details from the database
Track appointment booked with current time to mark it as booked if it is passed that time

How should this be manually tested?
Log in and go to  history you can switch between messages, voice calls and video calls where you will find what was discussed inform of either chats,video or voice calls depending on the screen you are on.
Navigate to the appointment screens and click on completed appointment to see the appointments which are passed the current time as they are considered completed you can also leave a review when the appointment has been completed by clicking on the leave review button

Any background context you want to provide?
N/A
